Backpack/Icon/Classes/BPKXlIconView.m

```diff
@@ -25,6 +25,17 @@ - (instancetype)initWithIconName:(nullable BPKXlIconName)iconName {
     return [super initWithIconName:iconName size:BPKIconSizeXLarge];
 }
 
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self) {
```

Comment:
This was missing from the XLIconView, but was present in the other 2 classes. As a result the icons were being rendered too small in the example app.

Reply:
Maybe it should be a protocol to enforce that it exists? Is that even a thing in Obj-C or is it just Swift?